apps/theming-designer/src/components/AccessibilityChecker.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { Card } from '@uifabric/react-cards';
+import { AccessibilityDetailsList } from './AccessibilityDetailsList';
+import { ITheme, IThemeRules, FabricSlots, BaseSlots } from '../../../../packages/office-ui-fabric-react/lib/index';
```

Comment:
Importing directly from '../../../../packages/office-ui-fabric-react' is causing build errors "import path(s) do not reference physical files. This can break AMD imports. Please ensure the following imports reference physical files" so I'm having to do the lib/index part.

Reply:
You shouldn't be doing cross-package relative imports. Just do `import { whatever } from 'office-ui-fabric-react/lib/index'` instead, and if that gives build errors, we'll help fix it.